ja_jp.lang

```diff
@@ -201,31 +201,75 @@ gui.xaero_arrow_scale=矢印の大きさ
 gui.xaero_arrow_colour=矢印の色
 gui.xaero_smooth_dots=ドットを滑らかにする
 gui.xaero_cave_zoom=洞窟でズームインする
+gui.xaero_safe_mode_box=デフォルトのOpenGLベース(GPU)の代わりに、このmodと同等なJavaベース(CPU)を使用します。 セーフモードはmodが機能しない場合のプランBです。セーフモードでは全ての機能を有効にしません。必要でない限り使わないでください。
 gui.xaero_height_limit=エンティティを表示する高度差上限
+gui.xaero_world_map=ワールドマップ
+gui.xaero_use_world_map=ワールドマップmodのチャンクを利用する
+gui.xaero_patron_capes=Patreonマント
 gui.xaero_entity_radar=エンティティ・レーダー
 gui.xaero_entity_info_potion_effects=ポーション効果（シングル限定）
 gui.xaero_entity_info_potion_effects_scale=ポーション効果の大きさ
 gui.xaero_terrain_depth=地形の立体表示
 gui.xaero_terrain_slopes=斜面の描画
 gui.xaero_block_transparency=ブロックの透過
 gui.xaero_team=チーム
+gui.xaero_world_server=ワールド/サーバー
+gui.xaero_subworld_dimension=サブワールド/ディメンション
+gui.xaero_switch_waypoint_set=ウェイポイントセットの切り替え
+gui.xaero_waypoint_opacity_ingame=ゲーム内ウェイポイントの透明度
+gui.xaero_waypoint_opacity_map=マップ上ウェイポイントの透明度
+gui.xaero_hide_world_names=IP/ワールド名を隠す
+gui.xaero_open_slime=キーを押して開く
 gui.xaero_options=設定
+gui.xaero_make_automatic=ワールド/サーバーを自動で作成する
+gui.xaero_make_automatic_msg1=よろしいですか？
+gui.xaero_make_automatic_msg2=選択されたワールド/サーバー間で全てのウェイポイントを入れ替えます。 このシミュレートは選択されたワールド/サーバーを自動化します。 何をしているのか理解してから実行してください。
```

Comment:
Seems wrong. I don't even understand the translation back. Should explain how this function swaps waypoints between 2 worlds/servers, one of which is currently the automatic one. After the swap, the non-auto world/server effectively becomes the automatic one.

Reply:
The second Japanese translated sentence was wrong. I`ll update it to match to the explanation that you commented